src/frontend/efiling-frontend/src/domain/package/package-review/PaymentList.js

```diff
@@ -68,7 +70,7 @@ export default function PaymentList({ payments }) {
                   payment.paymentCategory !== 1 && payment.paymentCategory !== 2
               )
               .map((payment) => (
-                <tr key={payment}>
+                <tr key={hash(payment)}>
```

Comment:
Found out javascript was trying to use [object Object] as the string value for a key, which is never unique.
Use a hash of the object instead